### PR TITLE
Gabriel Calero's fix (79.0/master)

### PR DESCRIPTION
**Pull request by NissimHadar**
# Description
Fixes an issue with broken build on Android

### CODE REVIEW

File: android/docker/Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN mkdir -p "$ANDROID_HOME" "$ANDROID_SDK_HOME" && \
     unzip sdk.zip && \
     rm sdk.zip && \
     yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
-
+RUN yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses && yes | $ANDROID_HOME/tools/bin/sdkmanager --update
 # Install Android Build Tool and Libraries
 RUN $ANDROID_HOME/tools/bin/sdkmanager --update
```
Comment (SamGondelman) [inline android/docker/Dockerfile]: This doesn't look right to me...these lines are repeated

File: android/docker/Dockerfile [hunk 2]
```diff
@@ -27,7 +27,7 @@ RUN mkdir -p "$ANDROID_HOME" "$ANDROID_SDK_HOME" && \
     unzip sdk.zip && \
     rm sdk.zip && \
     yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
```
Comment (amerhifi) [inline android/docker/Dockerfile]: as @SamGondelman mentioned this should be in that line, not duplicated

**Comment by conklin94122** [comment]
@SamGondelman when you are happy with the changes please approve then merge

File: android/docker/Dockerfile (hunk 1 above)
Comment (SamGondelman) [reply android/docker/Dockerfile]: I still don't think we need this line?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16611/HighFidelity-Beta-PR14750-a693e0b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16611/HighFidelity-Beta-Interface-PR14750-a693e0b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16611/HighFidelity-Beta-PR14750-a693e0b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16611/HighFidelity-Beta-Interface-PR14750-a693e0b.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16616/HighFidelity-Beta-PR14750-084d55a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16616/HighFidelity-Beta-PR14750-084d55a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16616/HighFidelity-Beta-Interface-PR14750-084d55a.dmg

File: android/docker/Dockerfile [hunk 3]
```diff
@@ -26,10 +26,9 @@ RUN mkdir -p "$ANDROID_HOME" "$ANDROID_SDK_HOME" && \
     curl -s -S -o sdk.zip -L "${SDK_URL}" && \
     unzip sdk.zip && \
     rm sdk.zip && \
-    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
-
+    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses && yes | $ANDROID_HOME/tools/bin/sdkmanager --update
+
```
Comment (SamGondelman) [inline android/docker/Dockerfile]: whitespace

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16616/HighFidelity-Beta-PR14750-084d55a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16616/HighFidelity-Beta-Interface-PR14750-084d55a.exe

File: android/docker/Dockerfile (hunk 2 above)
Comment (NissimHadar) [reply android/docker/Dockerfile]: Fixed

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16627/HighFidelity-Beta-PR14750-4edd072.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16627/HighFidelity-Beta-Interface-PR14750-4edd072.exe

**Pull request merged**